### DISCUSSION TITLE
Version on Documents website

### DISCUSSION THREAD

**Discussion by cwiggs** [seerr-team/seerr#1182] [Ideas]
I went through the docs and noticed postgres was an option so I tried to get postgres working with the most recent version of Jellyseerr. After some digging I noticed that the postgres option is only in the `develop` branch.

It would be nice to have the version of Jellyseer that the documents website is targeting, with a way to switch to different versions.

Thanks

**Comment by fallenbagel** [comment]
Postgres is not released yet. I'm not sure what you're saying about versions that match?
We literally just merged it in...

The docs got merged by that pr too... docs is always targeting latest. Usually we have a Tabs to differentiate. That PR just merged without the tabs.

**Reply by gauthier-th** [reply]
Docs is targeting latest, but is published by develop.
Here it doesn't make sense to have tabs for develop/latest, since Postgres will be merged soon.
It would be interesting to see what the docs was for latest, 1.9.2, etc., because people may run different version that the latest and wanting to see the docs for this version (e.g. no more armv7 support in v2, so stuck with v1.9.2)

**Reply by fallenbagel** [reply]
> It would be interesting to see what the docs was for latest, 1.9.2, etc., because people may run different version that the latest and wanting to see the docs for this version (e.g. no more armv7 support in v2, so stuck with v1.9.2)

That's possible too but then we will have to maintain docs for each release seperately

**Reply by gauthier-th** [reply]
> That's possible too but then we will have to maintain docs for each release seperately

Why? We could just keep the docs as it was the day of the release of the new version, so it's available as some kind of "archive" to show what was the content of the docs when the release was published.

**Reply by fallenbagel** [reply]
> > It would be interesting to see what the docs was for latest, 1.9.2, etc., because people may run different version that the latest and wanting to see the docs for this version (e.g. no more armv7 support in v2, so stuck with v1.9.2)
> 
> That's possible too but then we will have to maintain docs for each release seperately

I'm not talking about that. I'm talking about how docusaurus works. It reads from a single branch and builds to its own branch. And gh pages reads from that branch. I'm not sure to get the docs from the tags thats what I meant

**Reply by gauthier-th** [reply]
Oh i get it. We'll have to check if that's doable then

**Reply by cwiggs** [reply]
IMO versioning the docs would be best but also not releasing updated docs for the develop branch might help to?

**Reply by gauthier-th** [reply]
> IMO versioning the docs would be best but also not releasing updated docs for the develop branch might help to?

We only have one system to publish our docs. And this system build and publish docs from only one branch, hence the issue.
